src/ComposerHandler.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace JeroenG\Packager;
+
+use Closure;
+use RuntimeException;
+use Symfony\Component\Finder\Exception\DirectoryNotFoundException;
+
+trait ComposerHandler
+{
+    use ProcessRunner;
+
+    protected function removeComposerRepository($name)
+    {
+        return self::modifyComposerJson(function (array $composer) use ($name){
+            unset($composer['repositories'][$name]);
+            return $composer;
+        }, base_path());
+    }
+
+    /**
+     * Determines the path to Composer executable
+     * @todo Might not work on Windows
+     * @return string
+     */
+    protected static function getComposerExecutable(): string
```

Comment:
Why did you choose to make this static?

Reply:
Hmm. Mostly just for convenience and old habits.
The whole initial setup of the Testbench environment runs in a static context (`setupBeforeClass`), before Laravel has even been initialized. This is necessary because I can't switch the app's `base_path` once it's already up and running.
But honestly I should change it back to non-static, since I can just call the methods from the `$instance` variable. I just didn't notice earlier :)